### ISSUE TITLE
Export CSV container database column names instead of report column names

### ISSUE THREAD

**Issue by renaldrozario** [apache/superset#16750]
Export CSV contains database column names instead of report column names

This issue was previously reported here 2 years back
https://github.com/apache/superset/issues/5792
### Expected results

what you expected to happen.
Expected result is a CSV to show report column names
### Actual results
Currently it displays database column names

#### How to reproduce the bug

1. Go to Any report
2. Click on export to CSV
3. And the report shows database column name instead of report column name

### Environment

- superset version: `1.2.0`
- python version: `3.7`

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by junlincc** [comment]
- any feature flags active: ?

**Comment by renaldrozario** [comment]
Nope....

**Comment by junlincc** [comment]
@renaldrozario By Report, you mean Chart? 
what charts are you referring to? columns in downloaded CSV file are showing the verbose names. can you share a more detailed repro steps? thanks!

**Comment by mandeeplohan** [comment]
> * any feature flags active: ?

@junlincc  Sorry to bug you here. Can you help me with this https://github.com/apache/superset/issues/16749 
I shall be thankful to you :)

**Comment by renaldrozario** [comment]
@junlincc are you looking for more info?

**Comment by nld-8832** [comment]
@renaldrozario Hi, did you happen to find a solution/workaround for this? I'm running into the same particular problem.

**Comment by maudrid** [comment]
Yes, same issue for us. Our customer does not want to see the dataset column names.
They want to export the report they are looking at. 
Ant indication from devs if they agree that it is something that should be addressed?
